xwin.c

```diff
@@ -2348,12 +2376,36 @@ xwin_toggle_fullscreen(void)
 
 	g_fullscreen = !g_fullscreen;
 
+	// MultiMonitors : 1 monitor by default
+	g_num_monitors =1;
 	if (g_fullscreen)
 	{
 		x = 0;
 		y = 0,
 		width = WidthOfScreen(g_screen);
 		height = HeightOfScreen(g_screen);
+		
+#ifdef HAVE_XRANDR
+		// MultiMonitors : get geometry from xrand If we have
+		XRRScreenResources *xrrr = XRRGetScreenResources(g_display, DefaultRootWindow(g_display));
```

Comment:
What do you think about making this  separate function as you also use the same code in xwin.c +1987?

Reply:
done